natiami/assignment4/FindIslands.cpp

```diff
@@ -0,0 +1,45 @@
+#include"FindIslands.h"
+
+FindIslands::FindIslands(){}
+
+FindIslands::FindIslands(int nr_row, int nr_column, vector<vector<bool> >& mapOfTiles) : nr_row(nr_row), nr_column(nr_column), mapOfTiles(mapOfTiles){
+}
+
+bool FindIslands::checkBounds( int i, int j){

```

Comment:
Tip:
`checkBounds` is a good name, however you can give it a name that can be more read as a sentence, like `isInBounds` or `inBounds`:
```C++
if (inBounds(i,j)) {  // Read like a sentence: "if point (i,j) is in bounds then ... "
   // Do something with
}
```